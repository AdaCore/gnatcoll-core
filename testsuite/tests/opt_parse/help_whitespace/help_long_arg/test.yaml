driver: build_run_diff
description: Check that long argument name causes the previously default 25 column limit to expand
control:
    - [SKIP, "env.is_cross", "Tests using test.sh currently not supported on cross targets, see T616-039"]
