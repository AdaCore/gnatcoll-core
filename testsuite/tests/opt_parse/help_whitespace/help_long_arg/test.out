Help message column increased as expected